src/components/user-level.tsx

```diff
@@ -6,28 +6,32 @@ import { CircularProgress } from '@/components/ui/graphElements';
 
 import Widget from './ui/widget';
 
-interface ScoreProps {
-  score: number;
+interface UserLevelProps {
+  nextLevelPercentage: number;
   level: number;
 }
 
-const UserLevel = (scoreProps: ScoreProps) => {
-  const [score] = useState<number>(scoreProps.score);
-  const [level] = useState<number>(scoreProps.level);
+const UserLevel = (userLevelProps: UserLevelProps) => {
+  const [nextLevelPercentage] = useState<number>(
```

Comment:
Destructure the object could be a good idea here

```suggestion
const UserLevel = ({ nextLevelPercentage, level }: UserLevelProps) => {
```